mk/spksrc.service.use_alternate_tmpdir

```diff
@@ -21,15 +21,15 @@ service_clean_tmpdir ()
 {
   if [[ -e "${TMPDIR}" ]]; then
     if [ -n "$(ls -A ${TMPDIR})" ]; then
-       echo "Cleanup ${TMPDIR}"   >> "${INST_LOG}"
-       rm -rvf ${TMPDIR}/* >> "${INST_LOG}" 2>&1
```

Comment:
Is there any drawback to send activity logs to INST_LOG, maybe with recent DSM versions?

Reply:
Yes, it is not allowed to write to the ${INST_LOG} file anymore (except on DSM5).